inst/rmarkdown/templates/cholera_outbreak/skeleton/skeleton.Rmd

```diff
@@ -739,11 +743,12 @@ linelist_cleaned$age_group <- age_categories(linelist_cleaned$age_years,
 
 ## to combine different age categories use the following function 
 ## this prioritises the smaller unit, i.e. if given months and years, will return months first
-## generally, these delineations are NOT used for cholera
-
+## generally, these delineations are NOT used for measles.
```

Comment:
here too ...

Reply:
SUper weird! I don't remember making this change!